src/PattyServer.js

```
@@ -100,8 +101,7 @@ class PattyServer {
     try {
       process.setuid(this.options.processOwner);
     } catch(e) {
-      // throw PattyError.other()
-      this.logError(`Could not change process owner to "${this.options.processOwner}"`, e);
```

this is now before the init of the logger so we throw instead of logging